src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -1004,6 +1018,10 @@ interface GangMemberInfo {
   wantedLevelGain: number;
   /** Per Cycle Income for this gang member */
   moneyGain: number;
+  /** Per Cycle Exp object for each stat for this gang member returned as a object containing each stats value
+   * returns null in the even that the member does not have a given task.
```

Comment:
```suggestion
  /**
   * Per Cycle Exp object for each stat for this gang member.
   *
   * null in the event that the member does not have a given task.
```
* The second part of the sentence was redundant and confusing IMO
* In docs like this, you need a extra newline, otherwise it will all run together on one line when formatted. (Or, maybe that's what you want, but you will still want a period at the end of the prior sentence.)